docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
   services:
     build: dockerfiles/federation-demo
     ports:
-      - 4100:4100
+      - 4100:4000
```

Comment:
federation-demo is back on 4000? This is not great because people might want to run it without docker. How is this not breaking the CI?

Reply:
I think we aren't tracking the same submodule version. I'll make sure it's up to date, and check again.

Reply:
I've been able to reproduce it localy, after cleaning and rebuilding the docker images (`docker-compose up -d --build`).

Reply:
So much pain with this submodule :joy: maybe we should just pull the code to our repo...